net/openappid/Makefile

```diff
@@ -0,0 +1,52 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=openappid
+PKG_VERSION:=26425
+PKG_RELEASE:=1
+
+PKG_SOURCE:=snort-$(PKG_NAME).tar.gz
+PKG_SOURCE_URL:=https://www.snort.org/downloads/openappid/
+PKG_HASH:=8513877ce2264bb22119d911c2cf11f73735c866e2ca0d061c35eef6740d51f9
+
+PKG_MAINTAINER:=John Audia <thereadgraysky@proton.me>
+PKG_LICENSE:=GPL-2.0-only
+PKG_LICENSE_FILES:=COPYING
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/openappid
+  SUBMENU:=Firewall
+  SECTION:=net
+  CATEGORY:=Network
+  DEPENDS:=+snort3
+  TITLE:=Detection language and processing module for Snort
+  URL:=http://www.snort.org/
+  MENU:=1
+endef
+
+define Package/openappid/description
+  OpenAppId is an open, application-focused detection language and processing
+  module for Snort that enables users to create, share, and implement
+  application and service detection.
+endef
+
+define Build/Configure
+  true
+endef
+
+define Build/Compile
+  true
+endef
+
+define Package/openappid/install
+	$(INSTALL_DIR) $(1)/usr/lib/openappid/custom/{libs,lua,port}
+	$(INSTALL_DIR) $(1)/usr/lib/openappid/odp
+	$(HOST_TAR) -x -C $(1)/usr/lib/openappid/odp --strip-components=1 -f $(DL_DIR)/$(PKG_SOURCE)
```

Comment:
This is quite unusual to see it in install define.

Reply:
The upstream tarball untars to `/odp` not `tarballname-tarballver/odp` so this is the work-around someone on the forum helped me to use.  I am open to alternative suggestions.

Reply:
Basically we prefer copying files from `PKG_BUILD_DIR` or `PKG_INTALL_DIR`.
The tarball is already extracted by the buildroot at `Build/Prepare` stage, so you won't need to do it here again.

To copy files from PKG_BUILD_DIR, the following options need to be added (below line 20):
```
TAR_OPTIONS+= --strip-components 1
TAR_CMD=$(HOST_TAR) -C $(1) $(TAR_OPTIONS)
```

Reply:
The expected dirtree for this to work is:
```
/usr/lib/openappid/custom/{lua,port,libs} (all empty)
/usr/lib/openappid/odp/ (contents of tarball)
```

The changes above seem to untar the tarball to `./build_dir/target-aarch64_cortex-a72_musl/openappid-26425`

Reply:
I ended up inserting your code under line 20 and adding the calls to `install` as shown.  Please comment if bad.